docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,184 @@
+
+The fields available to fill at this stage are:
+
+- A text field with the **Scenario name**. 
+- A tag field to indicate which applications this scenario is designed to track. This will allow to create or reuse existing tags from other defined tracking scenarios.
+- A free text field for a **description** of the tracking scenario.
+
+![](images/ts-3.png)
+
+Clicking the **'Save'** button will create the tracking scenario. If you want to create multiple scenarios at once, for example, different scenarios for mobile and web tracking, click the **'Save & add another'** button instead. You can cancel the creation process by clicking **'Cancel'** button.
+
+When you're done creating your first tracking scenario, you will see a list of tracking scenarios on the page.
+
+![](images/ts-4.png)
+
+:::info
+By default, a newly created tracking scenario will have its status set as **Draft**. We'll explain how this can be changed and its purpose later.
+:::
+
+* * *
+
+## Editing a Tracking Scenario
+
+To edit an existing tracking scenario, start at the Data Structure and **Tracking scenarios** tab that contains the tracking scenario you want to edit. Click on the name of the tracking scenario you wish to edit.
+
+A new page will display the details of the tracking scenario. 
+
+![](images/ts-5.png)
+
+You can edit the following fields in this page.
+
+### Scenario Information
+
+Click the **'Edit information'** button in the **Scenario Information** section or the **pencil icon** at the right side of the tracking scenario name to open a dialog box view similar to the creation one, but with the current information already filled in.
+
+You will be able to modify all of the fields set previously, during the creation stage.
+
+To confirm the changes, click **'Save and update scenario'**. To cancel, click the **'Cancel'** button, both at the bottom of the dialog box.
+
+### Triggers
+
+Triggers describe situations in which tracking will be performed, such as button clicks, page views, app launches, in-app purchases, user interactions, login attempts, system events and more. These triggers are essential for developers when implementing tracking.
+
+Add multiple triggers by clicking the **'+ Add trigger'** button on the **Scenario Triggers** section. A new dialog box view will appear with a text field where you can specificy information about when, where and how the scenario will be triggered.
+
+![](images/ts-6.png)
+
+Once completed, you can confirm the changes by clicking the **'Save and update scenario'** button or cancel the changes by clicking the **'Cancel'** button.
+
+You will be able to edit the scenario after adding a trigger.
+
+### Implementation rules
+
+With implementation rules, you can define constraints and conditions on basic fields, based on the underlying schema properties of a particular tracking event. This way, you can ensure that once the events flows downstream through the pipeline, they will comply with the underlying schema.
```

Comment:
```suggestion
With implementation rules, you can define constraints and conditions on basic fields, based on the underlying schema properties of a particular tracking event. This way, you can ensure that once the events flow downstream through the pipeline, they will comply with the underlying schema.
```

Still unsure — aren’t we already ensuring that events comply with their respective schemas? :) Maybe an example here would help.

Reply:
updated